hapi-fhir-storage/src/main/java/ca/uhn/fhir/jpa/subscription/model/CanonicalSubscription.java

```diff
@@ -73,6 +74,8 @@
 	private String myPayloadSearchCriteria;
 	@JsonProperty("partitionId")
 	private Integer myPartitionId;
+	@JsonProperty("mustSendDeleteMessages")
+	private boolean myMustSendDeleteMessages;
```

Comment:
This name seems overly complicated.  Maybe something like 
```suggestion
	private boolean myIncludeDeleteMessages;
```